presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -10,16 +10,102 @@ it does not cause running queries to fail; instead new queries become queued.
 A resource group may have sub-groups or may accept queries, but may not do both.
 
 The resource groups and associated selection rules are configured by a manager, which is pluggable.
-Add an ``etc/resource-groups.properties`` file with the following contents to enable
-the built-in manager that reads a JSON config file:
+
+There are two built-in managers: file-based and database-based. Add an ``etc/resource-groups.properties``
+file with the appropriate configurations to enable one of the two managers.
+
+File Resource Group Manager
+---------------------------
+
+This manager reads the resource group configuration from a JSON config file.
 
 .. code-block:: none
 
     resource-groups.configuration-manager=file
     resource-groups.config-file=etc/resource-groups.json
 
-Change the value of ``resource-groups.config-file`` to point to a JSON config file,
-which can be an absolute path, or a path relative to the Presto data directory.
+* ``resource-groups.config-file``: a JSON file storing the configurations, which can be an absolute
+  path, or a path relative to the Presto data directory.
+
+Db Resource Group Manager
+-------------------------
+
+This manager periodically loads the resource group configuration from a database.
+
+.. code-block:: none
+
+    resource-groups.configuration-manager=db
+    resource-groups.config-db-url=jdbc:mysql://localhost:3306/config_db?user=presto_admin
+    resource-groups.max-refresh-interval=1m
+    resource-groups.exact-match-selector-enabled=true
+
+* ``resource-groups.config-db-url`` (required): Database URL to load configuration from.
+* ``resource-groups.max-refresh-interval``: Time period for which the cluster will continue to accept
+  queries after refresh failures cause configuration to become stale. The default is ``1h``.
+* ``resource-groups.exact-match-selector-enabled``: Boolean flag to enable usage of a selector requiring
+  an exact match of environment, source and query type.
+
+The configuration is loaded every second from the following tables under the specified db url. If the
+configuration ``resource-groups.exact-match-selector-enabled`` is set to ``true``, an additional table
+is used for loading exact-match based selectors. If any of the tables do not exist, Presto creates them
+on server startup.
+
++---------------------+-------------------------------------------------------+---------------------------------+
```

Comment:
This is way too much info for a table. I suggest we break this up into a separate section with proper code blocks and paragraphs that details more info.